packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -356,40 +412,81 @@ describe.skip("IntervalCollection fuzz testing", () => {
         }
     });
 
-    it("with default config", async () => {
-        const numClients = 3;
-
-        const containerRuntimeFactory = new MockContainerRuntimeFactory();
-        const sharedStrings = Array.from({ length: numClients }, (_, index) => {
-            const dataStoreRuntime = new MockFluidDataStoreRuntime();
-            const sharedString = new SharedString(
-                dataStoreRuntime,
-                String.fromCharCode(index + 65),
-                SharedStringFactory.Attributes,
-            );
-            const containerRuntime = containerRuntimeFactory.createContainerRuntime(dataStoreRuntime);
-            const services: IChannelServices = {
-                deltaConnection: containerRuntime.createDeltaConnection(),
-                objectStorage: new MockStorage(),
+    function runTests(seed: number, generator: Generator<Operation, FuzzTestState>, loggingInfo?: LoggingInfo): void {
+        it(`with default config, seed ${seed}`, async () => {
+            const numClients = 3;
+            const filepath = path.join(directory, `${seed}.json`);
```

Comment:
Not that there is any shortage of helpers in this file, but you could factor this out into a `getPath(seed)` - I like to do that for directories/file paths to be extra safe since a typo means you might make or delete unexpected files on your PC.